sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -49,29 +55,41 @@ export class NotificationActionsService {
     auditUserId: number,
     entityManager?: EntityManager,
   ): Promise<void> {
-    const templateId = await this.notificationMessageService.getTemplateId(
-      NotificationMessageType.StudentFileUpload,
-    );
-    const notificationToSend = {
-      userId: notification.userId,
-      messageType: NotificationMessageType.StudentFileUpload,
-      messagePayload: {
-        email_address: this.gcNotifyService.ministryToAddress(),
-        template_id: templateId,
-        personalisation: {
-          givenNames: notification.firstName ?? "",
-          lastName: notification.lastName,
-          dob: getDateOnlyFormat(notification.birthDate),
-          applicationNumber: notification.applicationNumber,
-          documentPurpose: notification.documentPurpose,
-          date: this.getDateTimeOnPSTTimeZone(),
+    const notificationDetails =
+      await this.notificationMessageService.getNotificationMessageDetails(
```

Comment:
For `getNotificationMessageDetails` we can maybe enforce the return type using the below.
```ts
Promise<Pick<NotificationMessage, "templateId" | "emailContacts">>
```
And consume the result as below.
```ts
const { templateId, emailContacts } =
  await this.notificationMessageService.getNotificationMessageDetails(
    NotificationMessageType.InstitutionAddsPendingOfferingNotificationForMinistry,
  );
```

Reply:
Are we not changing the line 58 to use `const { templateId, emailContacts }`?